modules/media/ajax/FileDownload.php

```diff
@@ -17,7 +17,7 @@
 $user =& User::singleton();
 //NOTE Should this be 'media_read' instead? It seems that downloading files
 //should be a read permission, not write.
-if (!$user->hasPermission('media_write')) {
+if (!$user->hasPermission('media_read')) {
```

Comment:
hey @KLaFleur this bit of code snuck in from your other PR. you might also need to run `npm run lint:php` on this PR branch so that this PR passes testing